src/components/toggle-button/assistant-departure-toggle-button.tsx

```diff
@@ -0,0 +1,80 @@
+import Link from 'next/link';
+import style from './assistant-departure-toggle-button.module.css';
+import { PageText, TranslatedString, useTranslation } from '@atb/translations';
+import { useState } from 'react';
+import { Typo } from '../typography';
+import { AnimatePresence, motion } from 'framer-motion';
+
+type AssistantDepartureTogglProp = {
+  href?: string;
+};
+
+type LinkProps = {
+  name: string;
+  title: TranslatedString;
+  link: string;
+};
+
+const links: LinkProps[] = [
+  {
+    name: 'assistant',
+    title: PageText.Assistant.title,
+    link: '/assistant',
+  },
+  {
+    name: 'departures',
+    title: PageText.Assistant.search.buttons.find.title,
+    link: '/departures',
+  },
+];
+
+const setInitialActiveLink = (href: string | undefined) => {
+  return href ? links[1].name : links[0].name;
```

Comment:
Should this check the `links` array for matching href and link instead of indexing it?

Reply:
I the href is defined as an attribute, the initial href should be links[1].name ("/departures"). If else, it should be "/assistant". 

That this logic is perhaps unnecessarily difficult and can most likely be written in a more intuitive way.

Reply:
This is usually a very good point and something to keep in mind every time you use `key` attribute. I think, however, as this uses static object this isn't a big issue here, as index will never change